scripts/dendrite_sytest.sh

```diff
@@ -61,12 +61,14 @@ fi
 
 # Check for new tests to be added to the test whitelist
 /src/show-expected-fail-tests.sh /logs/results.tap /src/sytest-whitelist \
-    /src/sytest-blacklist > /work/show_expected_fail_tests_output.txt || TEST_STATUS=$?
+    /src/sytest-blacklist | tee /work/show_expected_fail_tests_output.txt || TEST_STATUS=$?
 
 echo >&2 "--- Copying assets"
 
 # Copy out the logs
 rsync -r --ignore-missing-args --min-size=1B -av /work/server-0 /work/server-1 /logs --include "*/" --include="*.log.*" --include="*.log" --exclude="*"
+find /logs | xargs -r chmod go+r
+find /logs -type d | xargs -r chmod go+x
```

Comment:
Why do they need to be executable? Surely read-only is enough?

Reply:
For what it's worth, you can also use `chmod go+rX` where capital `X` only applies to directories, so you wouldn't need to do this in two separate steps.

Reply:
@kegsay the directories need +rx, as per POSIX listing access.

@reivilibre ah, nice! Done.